src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class SigningVerifyingX509Cert
+    {
+        const string ExampleXml = @"<?xml version=""1.0""?>
+<example>
+<test>some text node</test>
+</example>";
+
+        private static void SignXml(XmlDocument doc, AsymmetricAlgorithm key)
+        {
+            var signedXml = new SignedXml(doc)
+            {
+                SigningKey = key
+            };
+
+            var reference = new Reference();
+            reference.Uri = "";
+
+            reference.AddTransform(new XmlDsigEnvelopedSignatureTransform());
+
+            signedXml.AddReference(reference);
+
+            signedXml.ComputeSignature();
+            XmlElement xmlDigitalSignature = signedXml.GetXml();
+            doc.DocumentElement.AppendChild(doc.ImportNode(xmlDigitalSignature, true));
+        }
+
+        private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate)
+        {
+            XmlDocument xmlDoc = new XmlDocument();
+            xmlDoc.PreserveWhitespace = true;
+            xmlDoc.LoadXml(signedXmlText);
+
+            SignedXml signedXml = new SignedXml(xmlDoc);
+            var signatureNode = (XmlElement)xmlDoc.GetElementsByTagName("Signature")[0];
+            signedXml.LoadXml(signatureNode);
+            return signedXml.CheckSignature(certificate, true);
```

Comment:
Perhaps add a comment saying why we're setting `true` here.  Like:

// This test bypasses certificate verification because there's no automatic effective date
// functionality available.  Without bypassing verification the test only works until the certificate
// expires, and it's always unfortunate to have tests start failing just because the clock has moved.
//
// For real applications you should consider letting verification be applied, or providing verification
// in addition to (or in stead of) the builtin verification process.